app/dashboard/src/globals.d.ts

```diff
@@ -111,6 +111,30 @@ interface ProjectManagementApi {
   readonly setOpenProjectHandler: (handler: (projectInfo: ProjectInfo) => void) => void
 }
 
+// ========================
+// === File Browser API ===
+// ========================
+
+/** `window.fileBrowserApi` is a context bridge to the main process, when we're running in an
```

Comment:
Hope that we'll eventually collect all these API under a single field to avoid potential mismatches with the new API/extensions/even ids(https://stackoverflow.com/questions/3434278/do-dom-tree-elements-with-ids-become-global-properties)

Reply:
Ideally it'd be great to use `Symbol` for that

Reply:
> not possible, all our APIs are suffixed with Api. although it is a somewhat legitimate concern it's unlikely to happen in practice

Oh, how many times I've heard this 😭

It'd be great to keep all electron-related APIs under a single object. At least it'll be much easier to track that API

Reply:
i mean, personally i've never ever seen a spec that defined a global containing `Api` in its name, but i could be entirely wrong.

obviously it's preferable for everything to be under a single variable, i'm just pointing out that realistically i don't think it will be an issue for the next couple years at least.

Reply:
Yeah, currently it's not an issue and we can live with that, I just wish things were different 😊

Reply:
yeah definitely. i think this kind of thing is out of scope of this PR, i guess things like this should be opened as issues?

Reply:
opene as https://github.com/enso-org/enso/issues/10699